### DISCUSSION TITLE
user abusing bot to harass others

### DISCUSSION THREAD

**Discussion by girlmaya** [LuisMayo/ace-attorney-twitter-bot#42] [User reports and Twitter community problems]
take a look at [@ShinyLegendary's use of the bot](https://twitter.com/search?q=%40aceCourtBot%20from%3A%40ShinyLegendary&src=typed_query&f=live). she is making perhaps 10 requests per hour today. i don't know if you blacklist users but it would be ideal in this situation

**Comment by LuisMayo** [comment]
Moving it to a discussion

**Comment by LuisMayo** [comment]
After further review I can't really agree that what this user is doing is harassment. It doesn't seem targeted to any specific person or group.
It may be abusing the bot a bit. But given that the bot has no queue currently I'm not sure I'll take any actions.
